x-pack/plugins/apm/public/components/fleet_integration/apm_custom_assets_extension.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  CustomAssetsAccordionProps,
+  CustomAssetsAccordion,
+} from '../../../../fleet/public';
+import { useApmPluginContext } from '../../context/apm_plugin/use_apm_plugin_context';
+
+export function ApmCustomAssetsExtension() {
+  const { core } = useApmPluginContext();
+  const basePath = core.http.basePath.get();
```

Comment:
Right now this `core` value comes back as `undefined` so we wind up with an error here.

Reply:
should the UI extension be under the APM context, I know security solution is doing it like this  with a lazy too to avoid bringing too much stuff to the fleet bundle https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/lazy_endpoint_policy_edit_extension.tsx

Reply:
If you're using this in another plugin `useApmPluginContext` won't work but `useKibana` will, since APM uses that as well.

Reply:
Thanks @smith  - was able to get these set up cleanly with `useKibana`